.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           - ubuntu-latest
           - macos-latest
         ocaml-compiler:
-          - 4.10.2
```

Comment:
Use the latest compatible OCaml version at all times

Reply:
This one is a monorepo, so it actually has to be 4.10.2 specifically (for the expect tests).